wchat/screens/ContactList/components/ContactCell/index.js

```diff
@@ -0,0 +1,59 @@
+import React, { Component } from 'react';
+import { AppRegistry, StyleSheet, Text, Image, View } from 'react-native';
+
+export default class ContactCell extends Component {
+  render() {
+    return (
+      <View style={styles.container}>
+        <View style={styles.content}>
+          <Image style={styles.avatar} source={{uri: this.props.avatar}} />
+          <View style={styles.info}>
+            <Text style={styles.name}>{this.props.username} </Text>
+            <Text style={styles.message}>Lorem ipsum dolor sit amet...</Text>
+          </View>
+        </View>
+        <View style={styles.separator} />
+      </View>
+    );
+  }
+}
+
+const styles = StyleSheet.create({
+  container: {
+   flex: 1,
+  },
+  content: {
+   flex: 1,
+   flexDirection: 'row',
+   marginVertical: 12,
+  },
+  avatar: {
+    height: 44,
+    width: 44,
```

Comment:
We extract into a constant, sizes that are a prop of  a element. So, this image in particular use 44 pixels size height and width, so ill call avatarSize at the top of the styles.js... in this way if you have to set his size property, just change this const